### PR TITLE
Fix: Fixed group by date

### LINKED ISSUE #11333: Groups in the Downloads folder are not being sorted
### Description

On the latest stable version of Files (currently v2.4.33.0), sorting within file groups in the Downloads section works properly, but the grouping itself isn't.

For instance, I set both "group by" and "sort by" to be from newest to oldest, and the sorting within groups works great, but the "Older" group still always shows first, which is super inconvenient.

example

### Steps To Reproduce

1. Click on the sort icon in the Downloads folder,
2. Group By Date modified + Descending,
3. Groups are still ordered the wrong direction.

### Steps to fix

Haven't found a way to make it the correct sorting direction, so can't list any steps to fix.

### Files Version

Version: v2.4.33.0 OS Version: 10.0.22623.1255

### Windows Version

Windows 11 Pro 22H2 22623.1255

### Log file

[debug.log](https://github.com/files-community/Files/files/10761466/debug.log)

### LINKED ISSUE #11841: Bug: Formula of Earlier This Week & Last Week
### Description

The file I have marked with a red box appears in the wrong relative timeframe group.
( I've not looked at the source code ) It appears that the formula for, earlier this week = ( current date - 7 days ) or ( Sun to Sat ). Rather than, earlier this week = Mon to Sunday. Which is why I'm seeing the file appear in the wrong group. In File Explorer, the formula is the latter so the file appears in the Last Week group instead of Earlier This Week, which is correct.

( Not sure if including this is relevant to solving the issue - I don't think it is but I'll mention it anyway ) 
In Files, the sort by and group by are set to: 
Sort By = Date Modified, Descending. 
Group By = Date Modified, Descending.

**Figure 1 - Files**
Files_0YpvObvKHB

**Figure 2 - File Explorer**
8vwCwR62Vl

### Steps To Reproduce

Go to the Downloads.

### Requirements

Please change the formula for last week and earlier this week.

### Files Version

2.4.40.0

### Windows Version

10.0.19045.2728

### Log file

[debug.log](https://github.com/files-community/Files/files/11070366/debug.log)

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11841 
   Closes #11333  

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [X] Did you check for accessibility? You can use Accessibility Insights for this.
- [X] Did you remove any strings from the en-us resource file?
   - [X] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

**Details**
* This week and last week are now determined based on the current locale settings
* Added "Future" category for future timestamps
* Split "Older" category by year

### CODE REVIEW

File: src/Files.App/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -1446,29 +1446,35 @@
...
     <value>Tiny</value>
   </data>
+  <data name="Future" xml:space="preserve">
+    <value>Future</value>
+  </data>
   <data name="Today" xml:space="preserve">
     <value>Today</value>
   </data>
-  <data name="ItemTimeText_Yesterday" xml:space="preserve">
+  <data name="Yesterday" xml:space="preserve">
```
Comment (ferrariofilippo) [inline src/Files.App/Strings/en-US/Resources.resw:1455]: Can you update the resource names in other files too (only if you haven't changed the value)? That way we won't need to translate them again

**Review by ferrariofilippo — COMMENTED** [review]
Resources aside, LGTM

File: src/Files.App/Strings/en-US/Resources.resw (hunk 1 above)
Comment (hishitetsu) [reply src/Files.App/Strings/en-US/Resources.resw:1455]: 👍

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**